### DISCUSSION TITLE
How to remove - "Request may not be safe" waning for security providers ?

### DISCUSSION THREAD

**Discussion by dev-ramverma** [MetaMask/snaps#2025] [Q&A] [answered]
Can anyone help me to remove the security provider warning?
![image](https://github.com/MetaMask/snaps/assets/118476064/484f856e-1307-4a52-851a-cf4984ff25be)

Error - 

Request may not be safe
Because of an error, this request was not verified by the security provider.Proceed with caution

**Comment by Montoya** [comment, marked as answer]
This is not a Snaps-related question. I would recommend opening an issue in the extension repo instead: https://github.com/MetaMask/metamask-extension/issues
